libsolidity/analysis/StaticAnalyzer.cpp

```diff
@@ -90,6 +90,46 @@ void StaticAnalyzer::endVisit(FunctionDefinition const&)
 	m_localVarUseCount.clear();
 }
 
+bool StaticAnalyzer::visit(ModifierInvocation const& _modifier)
+{
+	if (!m_constructor)
+		return true;
+
+	bool const v050 = m_currentContract->sourceUnit().annotation().experimentalFeatures.count(ExperimentalFeature::V050);
+
+	if (auto contract = dynamic_cast<ContractDefinition const*>(_modifier.name()->annotation().referencedDeclaration))
+		for (auto const &currentContract: m_currentContract->annotation().linearizedBaseContracts)
```

Comment:
`auto const& curr...`

Reply:
Also `currentContract` and `m_currentContract` are named similarly but have different meaning. What about `base`?